src/core/plugins/oas3/components/request-body-editor.jsx

```diff
@@ -116,14 +116,22 @@ export default class RequestBodyEditor extends PureComponent {
         <div className="body-param-options">
           <div className="body-param-edit">
             {
-              !isExecute ? null
-                         : <Button className={isEditBox ? "btn cancel body-param__example-edit" : "btn edit body-param__example-edit"}
-                                   onClick={this.toggleIsEditBox}>{ isEditBox ? "Cancel" : "Edit"}
-                           </Button>
-
+              !isExecute
+                ? null
+                : <Button
+                    className={`sui-btn sui-btn--secondary body-param__example-edit ${isEditBox ? "cancel" : "edit"}`}
+                    onClick={this.toggleIsEditBox}
+                  >
+                    <span>{isEditBox ? "Cancel" : "Edit"}</span>
+                  </Button>
             }
             { userDidModify &&
-              <Button className="btn ml3" onClick={() => { this.resetValueToSample(mediaType) }}>Reset</Button>
+              <Button
+                className="sui-btn sui-btn--secondary ml3"
```

Comment:
**Change request**: drop `ml3` (it's a [Tachyons class](http://tachyons.io/docs/layout/spacing/) that I really shouldn't have kept around, cc #4460), set a `margin-left` for the button as needed instead